code/modules/paperwork/paper.dm

```diff
@@ -29,6 +29,18 @@
 	var/list/stamped
 	var/rigged = 0
 	var/spam_flag = 0
+	var/contact_poison // Reagent ID to transfer on contact
```

Comment:
That would make it basically chemist toy. With is even more useless than it already is.